_posts/2016-04-05-todo-track-at-collab-summit.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "TODO Track at the LF Collaboration Summit"
+author: todogroup
+---
+
+Last week, the TODO Group led a track at the [Collaboration Summit](http://events.linuxfoundation.org/events/collaboration-summit) which featured a variety of talks dedicated to open source program management:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">today we are happy to host the TODO (open source program management) track at <a href="https://twitter.com/hashtag/lfcollab?src=hash">#lfcollab</a> today, we&#39;ll be tweeting! <a href="https://t.co/h8zOxn4D9F">pic.twitter.com/h8zOxn4D9F</a></p>&mdash; TODO Group (@todogroup) <a href="https://twitter.com/todogroup/status/715214451830460416">March 30, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Listed below are some of slides from those talks:
+
+Gil Yehuda ([@gyehuda](https://twitter.com/gyehuda)) runs the Open Source Office at Yahoo and presented a talk highlighting some of the governance issues facing corporate based open source program offices. Gil made the case that medium to large tech companies needs an Open Source Program Office. The tech industry acknowledges the ubiquity of open source in the corporate tech stack as well as in products, yet companies face governance issues that need to be addressed properly, consistently, and with goals in mind. He noted that Jim Zemlin lamented at the event keynote that only about 30% of the tech companies surveyed had basic Open Source Governance policies in place, let alone a mature process to manage their involvement in Open Source. The industry needs to improve this, since corporate contributions to Open Source are an increasingly important part of the ecosystem.
```

Comment:
Thanks, we will do a copy edit when we have slides posted.

Having a common lexicon amongst ourselves would be useful.